### PR TITLE
The id names of few activities have been changed to suit the Views in the XML file

### LINKED ISSUE #706: fix: senseless names for ids in activity_login
**Summary:**
<TextView
            android:id="@+id/textView2"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:layout_marginStart="0dp"
            android:layout_marginLeft="0dp"
            android:layout_marginEnd="@dimen/marginBetweenWords"
            android:layout_marginRight="@dimen/marginBetweenWords"
            android:text="@string/don_t_have_an_account"
            android:textColor="@color/colorTextSecondary" />

       <TextView
            android:id="@+id/textView"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:text="@string/create_one"
            android:textColor="@color/colorAccent" />

These views have senseless ids and ids could be changed according to it's purpose.

### PR DESCRIPTION
**Pull request by haran2248**
The inappropriate ids are changed to appropriate ones without changing sizes

## Issue Fix
Fixes #706 

## Description

The blunt android ids have been changed to meaningful ids

##

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: mifospay/src/main/res/layout/fragment_home.xml [hunk 1]
```diff
@@ -65,7 +65,7 @@
 
             <ImageView
                 android:layout_gravity="center_horizontal"
-                android:id="@+id/imageView"
+                android:id="@+id/shape_bottom_sheet_dialog_grip"
```
Comment (Bhavnaharitsa) [inline mifospay/src/main/res/layout/fragment_home.xml:68]: could be something better

**Comment by Bhavnaharitsa** [comment]
@haran2248 Also, please follow the PR guidelines which I guess you can find in the Readme.md.

File: mifospay/src/main/res/layout/fragment_home.xml (hunk 1 above)
Comment (haran2248) [reply mifospay/src/main/res/layout/fragment_home.xml:68]: Um.. what should I name I the drawable name was that so I thought I would be good if it was named the same

**Comment by haran2248** [comment]
Um.. what should I name it? I thought it would be good if the Id name and drawable name was the same

**Comment by Bhavnaharitsa** [comment]
> Um.. what should I name it? I thought it would be good if the Id name and drawable name was the same

Something more sensible, I feel. Also change the PR title

File: mifospay/src/main/res/layout/activity_login.xml [hunk 2]
```diff
@@ -93,7 +93,7 @@
             android:textColor="@color/colorTextSecondary" />
 
         <TextView
-            android:id="@+id/textView"
+            android:id="@+id/create_account"
```
Comment (devansh-299) [inline mifospay/src/main/res/layout/activity_login.xml]: Please change this to `tv_create_account`

File: mifospay/src/main/res/layout/activity_merchant_transaction.xml [hunk 3]
```diff
@@ -46,7 +46,7 @@
                 android:orientation="vertical">
 
                 <ImageView
-                    android:id="@+id/imageView"
+                    android:id="@+id/bottom_sheet_dialog_grip"
```
Comment (devansh-299) [inline mifospay/src/main/res/layout/activity_merchant_transaction.xml]: Change this to something like `iv_bottom_sheet_grip`

File: mifospay/src/main/res/layout/activity_mobile_verification.xml [hunk 4]
```diff
@@ -18,7 +18,7 @@
             <android.support.v7.widget.Toolbar
                 xmlns:android="http://schemas.android.com/apk/res/android"
                 xmlns:app="http://schemas.android.com/apk/res-auto"
-                android:id="@+id/toolbar"
+                android:id="@+id/mobile_verification_toolbar"
```
Comment (devansh-299) [inline mifospay/src/main/res/layout/activity_mobile_verification.xml]: The ids of views should ideally follow  \<what>\<where>\<description> pattern. Hence for this view you should first mention `toolbar` and then the description

File: mifospay/src/main/res/layout/content_bottom_sheet_account_details.xml [hunk 5]
```diff
@@ -7,7 +7,7 @@
     android:orientation="vertical">
 
     <ImageView
-        android:id="@+id/imageView"
+        android:id="@+id/shape_bottom_sheet_dialog_grip"
```
Comment (devansh-299) [inline mifospay/src/main/res/layout/content_bottom_sheet_account_details.xml]: Same here

File: mifospay/src/main/res/layout/fragment_home.xml (hunk 1 above)
Comment (devansh-299) [reply mifospay/src/main/res/layout/fragment_home.xml:68]: Yes, @haran2248 please follow the above pattern

**Review by devansh-299 — CHANGES_REQUESTED** [review]

**Comment by haran2248** [comment]
Ill work on this for sure

**Pull request merged**